.circleci/config.yml

```diff
@@ -39,7 +39,7 @@ jobs:
           name: Mock the storage classes
       - redhat-openshift/login-and-update-kubeconfig:
           insecure-skip-tls-verify: true
-          openshift-platform-version: 3.x
+          openshift-platform-version: 4.x
```

Comment:
Oooh is this new? It wasn't available until recently

Reply:
I don't know if it will work, but test-local-cluster is failing because it can't find the podDisruptionBudget in policy/v1. I figured it was because it was using platform version 3.x